extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -2549,7 +2551,13 @@ describes.realWin('no signing', {amp: true}, (env) => {
             a4a = new MockA4AImpl(a4aElement);
             consentString = 'test-consent-string';
             gdprApplies = true;
-            consentMetadata = {gdprApplies};
+            additionalConsent = 'abc123';
+            consentStringType = 1;
+            consentMetadata = {
```

Comment:
nit: can save a few lines of code by inlining `gdprApplies`, `addtionalConsent` and `consentStringType` value.